examples/storm-kafka-client-examples/pom.xml

```diff
@@ -42,7 +42,9 @@
             <groupId>org.apache.storm</groupId>
             <artifactId>storm-kafka</artifactId>
             <version>${project.version}</version>
+            <!-- You can reduce jar size by uncommenting this and providing the dependencies manually. See the README for details.
             <scope>${provided.scope}</scope>
```

Comment:
Yes, I somehow missed the dollar. I'll try reverting this and update the readme to set the scope to compile